matter/src/data_model/device_types.rs

```diff
@@ -78,7 +78,7 @@ pub const DEV_TYPE_ON_SMART_SPEAKER: DeviceType = DeviceType {
     drev: 2,
 };
 
-pub fn device_type_add_on_off_light(node: &mut WriteNode) -> Result<u32, Error> {
+pub fn device_type_add_on_off_light(node: &mut WriteNode) -> Result<EndptId, Error> {
```

Comment:
ClusterId?

Reply:
It’s expected to add a device type with all the relevant clusters into it